life_dashboard/tests/test_auth.py

```
@@ -0,0 +1,217 @@
+        response = self.client.get(url)
+        assert response.status_code == 302
+
+    def test_signup_page_selenium(self):
+        self.driver.get(f'{self.live_server_url}{reverse("dashboard:register")}')
+        assert "Register" in self.driver.page_source
+
+    def test_login_page_selenium(self):
+        self.driver.get(f'{self.live_server_url}{reverse("dashboard:login")}')
+        assert "Login" in self.driver.page_source
+
+    def test_register_user_selenium(self):
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)